### PR TITLE
Candlestick and OHLC chart

### PR DESCRIPTION
**Pull request by priyatharsan**
closes https://github.com/plotly/plotly.github.io/issues/424

### CODE REVIEW

File: _posts/financial/2017-03-08-Candlestick-Chart.md [hunk 1]
```diff
@@ -1,66 +1,60 @@
...
-      img: "![Timescale Buttons](../static/images/make-a-candlestick/timescale-buttons.png)"
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart, visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
+    - copy: "Use the 'General' section under the 'Style' menu to set the plot title, as well as change the layout background, margin color and font styles."
+    - copy: "To set the plot title, type the title text within the textbox provided under the 'Title' property."
+      img: "![Type title](../static/images/candlestick-chart/candlestick-title.png)"
+    - copy: "Another approach is to click and then enter the title directly on the plot interface."
+      img: "![Type directly title](../static/images/candlestick-chart/candlestick-title-direct.png)"
+    - copy: "Use the 'Traces' section under the 'Style' menu to change the properties of the trace such as the colorscale, lighting effects and its position, and hoverinfo."
```
Comment (ashmpace) [inline _posts/financial/2017-03-08-Candlestick-Chart.md]: Suggestion: "lighting effects, position and hoverinfo."

File: _posts/financial/2017-03-08-OHLC-Chart.md [hunk 2]
```diff
@@ -1,66 +1,60 @@
...
-      img: "![Timescale Buttons](../static/images/make-a-ohlc/timescale-buttons.png)"
+ - title: Style a Chart
+   sub-steps:
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart, visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
+    - copy: "Use the 'General' section under the 'Style' menu to set the plot title, as well as change the layout background, margin color and font styles."
+    - copy: "To set the plot title, type the title text within the textbox provided under the 'Title' property."
+      img: "![Type title](../static/images/ohlc-chart/ohlc-title.png)"
+    - copy: "Another approach is to click and then enter the title directly on the plot interface."
+      img: "![Type directly title](../static/images/ohlc-chart/ohlc-title-direct.png)"
+    - copy: "Use the 'Traces' section under the 'Style' menu to change the properties of the trace such as the colorscale, lighting effects and its position, and hoverinfo."
```
Comment (ashmpace) [inline _posts/financial/2017-03-08-OHLC-Chart.md]: Suggestion: "lighting effects, position, and hoverinfo."

File: _posts/financial/2017-03-08-Candlestick-Chart.md (hunk 1 above)
Comment (priyatharsan) [reply _posts/financial/2017-03-08-Candlestick-Chart.md]: replace with "lighting effects, light position and hoverinfo." ?

File: _posts/financial/2017-03-08-OHLC-Chart.md (hunk 2 above)
Comment (priyatharsan) [reply _posts/financial/2017-03-08-OHLC-Chart.md]: replace with "lighting effects, light position and hoverinfo." ?

**Comment by priyatharsan** [comment]
done! @ashmpace Thanks

**Comment by cldougl** [comment]
Looks like the images aren't showing up for these:

File: _posts/financial/2017-03-08-Candlestick-Chart.md [hunk 3]
```diff
@@ -1,66 +1,61 @@
...
-      img: "![Timescale Buttons](../static/images/make-a-candlestick/timescale-buttons.png)"
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart, visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
+    - copy: "Use the 'General' section under the 'Style' menu to set the plot title, as well as change the layout background, margin color and font styles."
+    - copy: "To set the plot title, type the title text within the textbox provided under the 'Title' property."
+      img: "![Type title](../static/images/candlestick-chart/candlestick-title.png)"
+    - copy: "Another approach is to click and then enter the title directly on the plot interface."
+      img: "![Type directly title](../static/images/candlestick-chart/candlestick-title-direct.png)"
+    - copy: "Use the 'Traces' section under the 'Style' menu to change the properties of the trace such as the colorscale, lighting effects, light position, and hoverinfo."
```
Comment (cldougl) [inline _posts/financial/2017-03-08-Candlestick-Chart.md]: the lighting options apply to 3d traces

File: _posts/financial/2017-03-08-Candlestick-Chart.md [hunk 4]
```diff
@@ -1,66 +1,61 @@
...
- - title: Create Chart
+ - title: Create a Chart
    sub-steps:
-    - copy: "After adding your own data, go to GRAPH on the left-hand side, then 'Create'. Click 'Chart Type', then choose 'Candlestick Chart' in the 'Financial' column."
-      img: "![Choose chart main](../static/images/make-a-candlestick/choose-chart-main.JPG)"
-    - copy: "On the left-hand side, in the trace panel, you can assign each variable. You should then fill out the X, Open, High, Low, and Close dropdown to create the plot."
-      img: "![Adding data](../static/images/make-a-candlestick/trace-panel.PNG)"
+    - copy: "After adding the data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Candlestick' under 'Financial' chart type."
+      img: "![Chart Type](../static/images/candlestick-chart/candlestick-chart-type.png)"
+    - copy: "Next, select 'X', 'Open', 'High', 'Low' and 'Close' values from the dropdown menus. This will create the candlestick trace as seen below."
```
Comment (cldougl) [inline _posts/financial/2017-03-08-Candlestick-Chart.md]: can you add a bit more information about this creating a trace with a different color for increasing vs decreasing

Comment (priyatharsan) [reply _posts/financial/2017-03-08-Candlestick-Chart.md]: 👍

**Comment by cldougl** [comment]
💃

**Pull request merged**